velox/core/Config.h

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <folly/Conv.h>
+#include <folly/Optional.h>
+#include <folly/Synchronized.h>
+#include <mutex>
+#include <set>
+#include <typeindex>
+#include <unordered_map>
+#include "velox/common/base/Exceptions.h"
+
+namespace facebook::velox {
+
+class Config {
```

Comment:
NYC: why config is not part of core namespace?

Reply:
Have no idea. :)
This is some old code.
I left it as is because there would be tons of build issues, including unfixable.